tsl/src/bgw_policy/job.c

```diff
@@ -73,9 +73,10 @@ log_retention_boundary(int elevel, PolicyRetentionData *policy_data, const char
 
 	if (OidIsValid(outfuncid))
 		elog(elevel,
-			 "%s \"%s\": dropping data older than %s",
+			 "%s \"%s\": dropping data %s %s",
```

Comment:
FWIW, "dropping data created before" is kind of misleading because data can be inserted into a chunk long after it is created. Actually this applies to the previous message as well.

Maybe we should just take the opportunity to tweak this message to "dropping chunks ...."